Src/FluentAssertions/Primitives/ReferenceTypeAssertions.cs

```diff
@@ -418,15 +418,18 @@ public AndConstraint<TAssertions> NotBeAssignableTo(Type type, string because =
     }
 
     /// <summary>
-    /// Asserts that the criteria provided by the element inspector is satisfied.
+    /// Allows combining one or more assertions using the other assertion methods that this library offers on an instance of <typeparamref name="T"/>.
     /// </summary>
-    /// <param name="expected">The element inspector which must be satisfied by the <typeparamref name="TSubject" />.</param>
+    /// <remarks>
+    /// If multiple assertions executed by the <paramref name="assertion"/> fail, they will be raised as a single failure.
+    /// </remarks>
+    /// <param name="assertion">The element inspector which must be satisfied by the <typeparamref name="TSubject" />.</param>
     /// <returns>An <see cref="AndConstraint{T}" /> which can be used to chain assertions.</returns>
-    /// <exception cref="ArgumentNullException"><paramref name="expected"/> is <see langword="null"/>.</exception>
-    public AndConstraint<TAssertions> Satisfy<T>(Action<T> expected)
+    /// <exception cref="ArgumentNullException"><paramref name="assertion"/> is <see langword="null"/>.</exception>
+    public AndConstraint<TAssertions> Satisfy<T>(Action<T> assertion)
```

Comment:
I think `assertion` is a bit misleading here 🤔

Reply:
You're supposed to use that action to build a composite assertion, so I think it's a suitable term.